package.json

```diff
@@ -27,7 +27,8 @@
     "chalk": "^4.0.0",
     "cosmiconfig": "^7.0.0",
     "create-jest-runner": "^0.11.2",
-    "dot-prop": "^6.0.1"
+    "dot-prop": "^6.0.1",
+    "find-up": "^5.0.0"
```

Comment:
let's please not add packages from this maintainer, since they're converting to ESM-only and make frequent breaking changes.

Reply:
I only added this because it's [what eslint uses](https://github.com/eslint/eslint/blob/d85efad655deacc0dc3fdbbace33307094c3b91b/lib/eslint/flat-eslint.js#L266-L271) and I think we should try to match the functionality as much as possible.  Ideally the `findFlatConfigFile` function (or even `shouldUseFlatConfig`](https://github.com/eslint/eslint/blob/d85efad655deacc0dc3fdbbace33307094c3b91b/lib/cli.js#L283-L301) would be exposed directly by eslint but it isn't, so we need to mimic it.

Reply:
Fair; pinning to v5 is fine i guess for now.

Regarding what eslint exposes, if we have a need for it, please file an issue on eslint, and they may be willing to expose it :-)

Reply:
Yep that's fair -- I'll put in an issue / PR to eslint and see if they're willing to do so.  Will keep this in this PR for now but if things move forward on that side, I'll either update this PR or put in a follow up.

Reply:
In case you're following along, eslint issue is here: https://github.com/eslint/eslint/issues/17129

I'll submit a PR over there if/when the eslint folks get back to me on which strategy would be acceptable.